### PR TITLE
ping: add .ping command

### LINKED ISSUE #1874: add ping/pong response to ping.py
### The problem
The default ping module doesn't include ping/pong despite it's name and I'd rather not replace it just for that (I could move it out of core hacks but why not upstream)!

### The solution
Add it, I will after I write this.

### Example

09:54:36 <+RhinosF1> .ping
09:54:37 <@ZppixBot-test> Pong!

### PR DESCRIPTION
**Pull request by RhinosF1**
resolves #1874

### Checklist
- [x] I have read [CONTRIBUTING.md](https://github.com/sopel-irc/sopel/blob/master/CONTRIBUTING.md)
- [x] I can and do license this contribution under the EFLv2
- [x] No issues are reported by `make qa` (runs `make quality` and `make test`)
- [x] I have tested the functionality of the things this change touches

### CODE REVIEW

File: sopel/modules/ping.py [hunk 1]
```diff
@@ -29,3 +29,9 @@ def rude(bot, trigger):
 @thread(False)
 def interjection(bot, trigger):
     bot.say(trigger.nick + '!')
+
+
+@commands('ping')
+def ping(bot, trigger):
+    """Reply to ping command."""
+    bot.say('Pong!')
```
Comment (Exirel) [inline sopel/modules/ping.py:37]:
```suggestion
    bot.reply('Pong!')
```

**Comment by RhinosF1** [comment]
> Why does this touch `sopel/irc/__init__.py`? ("Reviewing" as "Request changes" so it won't look ready-to-merge in the list.)

because my rebase failed

**Comment by RhinosF1** [comment]
Fixed my mess

**Comment by dgw** [comment]
@RhinosF1 Your "mess" isn't "fixed" just yet. Need to squash it down again.

**Comment by RhinosF1** [comment]
That’ll be fun.

**Pull request merged**